### PR TITLE
Adapt recipe to work with OpenStack Heat

### PR DESCRIPTION
**Pull request by MatMaul**
Cf comment of commits for details.

### CODE REVIEW

File: tasks/wn.yaml [hunk 1]
```diff
@@ -1,25 +1,4 @@
...
-    - shell: cat /etc/hosts | sed "/{{slurm_server_name}}/d" > /tmp/newhosts  
-    # Now we add a new entry with the new IP address
-    - shell: echo "{{ slurm_server_ip }} {{ slurm_server_name }}" >> /tmp/newhosts 
-    # Let's set the new /etc/hosts file
-    - shell: cp /tmp/newhosts /etc/hosts && rm /tmp/newhosts 
-
-    - service: name=firewalld state=stopped
-      when: ansible_os_family == "RedHat" and ansible_distribution_major_version|int >= 7
-      ignore_errors: yes
-
```
Comment (micafer) [inline tasks/wn.yaml]: Hi @MatMaul,
Why did you remove the avobe lines?

Comment (MatMaul) [reply tasks/wn.yaml]: yep I should put back the firewalld stopped task. However I don't really understand what the first lines are doing to the hosts file. Perhaps we don't need it anymore with https://github.com/indigo-dc/tosca-types/pull/138 ? I had no trouble without them.

Comment (micafer) [reply tasks/wn.yaml]: In principle with the PR referenced the first lines should be unnecessary. So please add the firewalld stopped task.

**Comment by MatMaul** [comment]
Should be ok now !

**Review by micafer — APPROVED** [review]
LGTM
